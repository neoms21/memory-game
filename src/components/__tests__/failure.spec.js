import React from 'react';
import Renderer from 'react-test-renderer';
import 'jest-styled-components';
import Failure from '../failure';

describe('Failure tests', () => {
  it('should render null', () => {
    const wrapper = Renderer.create(<Failure />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should render error', () => {
    const wrapper = Renderer.create(<Failure isGameOver={true} score={11} />);

    expect(wrapper).toMatchSnapshot();
  });
});
