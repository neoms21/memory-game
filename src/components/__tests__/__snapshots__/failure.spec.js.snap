// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Failure tests should render error 1`] = `
.c0 {
  background: #c42d2d;
  color: #fff;
  padding: 8px;
  margin-bottom: 10px;
}

<div
  className="c0"
  id="error"
>
  You lost the game!! you score is 
  11
</div>
`;

exports[`Failure tests should render null 1`] = `null`;
